Makefile

```diff
@@ -9,7 +9,7 @@ COMMON_SRC = common.c
 COMMON_OBJ := $(COMMON_SRC:.c=.o)
 COMMON_HEADERS = common.h
 
-ALL = switched threaded predecoded subroutined threaded-cached tailrecursive translated native
+ALL = switched threaded predecoded subroutined threaded-cached tailrecursive translated inline_translated native
```

Comment:
It is better be called `translated-inline` to match conventions used for the rest of VM names.